src/System.IO.IsolatedStorage/System.IO.IsolatedStorage.sln

```diff
@@ -18,6 +18,9 @@ EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.IO.IsolatedStorage", "ref\System.IO.IsolatedStorage.csproj", "{27225772-FE8B-49D7-8E58-29242D536130}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{8751BC6B-FD4E-40E0-A5F8-63F92D987719}"
+	ProjectSection(SolutionItems) = preProject
+		tests\System.IO.IsolatedStorage.Tests.builds = tests\System.IO.IsolatedStorage.Tests.builds
```

Comment:
I don't think we usually add the .builds files to the solution. I mean it doesn't hurt I guess, but it's just inconsistent with the rest of the repo.